### PR TITLE
EOS-13699 - Handle key corruption in recovery

### PR DESCRIPTION
**Pull request by shanbhagsv1**

### CODE REVIEW

File: s3recovery/s3recovery/s3recovercorruption.py [hunk 1]
```diff
@@ -53,8 +53,13 @@ def cleanup_bucket_metadata_entries(self, index_id):
         data_as_dict = self.parse_index_list_response(list_index_response)
         key_list = list(data_as_dict.keys())
         for key in key_list:
-            key_part = key.split('/')[1]
-            if key_part not in self.common_keys:
+            if (key.count('/') == 1):
```
Comment (amitwac2608) [inline s3recovery/s3recovery/s3recovercorruption.py:56]: use a constant for '/' like METADATA-INDEX-KEYNAME-SEPARATOR.

**Review by amitwac2608 — APPROVED** [review]
gave a minor comment.

File: s3recovery/s3recovery/s3recovercorruption.py (hunk 1 above)
Comment (shanbhagsv1) [reply s3recovery/s3recovery/s3recovercorruption.py]: Done

**Comment by shanbhagsv1** [comment]
Build Job http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server/27/

**Review by nileshgovande — APPROVED** [review]

**Pull request merged**